test/ConsoleLogger.test.js

```diff
@@ -39,6 +40,27 @@ test('Service result objects should be provided to consumer one by one', async t
 
   await passThrough
     .pipe(consoleLogger)
-    .on('data', data => t.deepEqual(data, serviceResults[count++]));
+    .on('data', data => {
+      if (count !== serviceResults.length) {
+        t.deepEqual(data, serviceResults[count++])
+      }
+    });
 });
 
+test('Totals should be output at the end', async t => {
+  const consoleLogger = new ConsoleLogger({objectMode: true});
+  const passThrough = new PassThrough({objectMode: true});
+  let count = 0;
+
+  serviceResults.forEach(serviceResult => passThrough.write(serviceResult));
+  passThrough.end();
+
+  await passThrough
+    .pipe(consoleLogger)
+    .on('data', data => {
+      if (count === serviceResults.length) {
+        t.is(data, '5 occurences found in 2 projects');
+      };
+      count++;
+    });
+});
```

Comment:
It feels like these 2 tests would feel nicer combined into 1 how about making a copy of `serviceResults` in the test and `.push('5 occurences found in 2 projects')` into it, then test against that?